### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'find'

### ISSUE THREAD

**Issue by wernight** [ccxt/ccxt#533]
In some cases when trying to cancel an order from Kraken, there is this exception:

```
>>> cancel_order(order_id, symbol)
...
  File ".../site-packages/ccxt/kraken.py", line 480, in cancel_order
    if message.find('EOrder:Unknown order') >= 0:
AttributeError: 'NoneType' object has no attribute 'find'
```

Looks like `message` might be None in `kraken.py` around line 480.

**Comment by kroitor** [comment]
@wernight thx for reporting this! Can you please paste verbose output of it (set `.verbose = True` on kraken instance) ?

**Comment by kroitor** [comment]
This issue is fixed as of 1.10.56+. Thx again for reporting!
